src/Raven.Server/Utils/Enumerators/TransactionForgetAboutCurrentPreviousRevisionEnumerator.cs

```diff
@@ -11,9 +11,35 @@ public TransactionForgetAboutCurrentPreviousRevisionEnumerator([NotNull] IEnumer
     {
     }
 
-    protected override void ForgetAbout((Document Previous, Document Current) item)
+    protected override void ForgetAbout(ForgetAboutItem item)
     {
-        DocsContext.Transaction.ForgetAbout(item.Current);
-        DocsContext.Transaction.ForgetAbout(item.Previous);
+        if (item.IsCloned)
```

Comment:
So we do it only for cloned? After reading the above comment:

> the clone is needed because we are going to forget about the _current document

I thought that we won't be forgetting about cloned ones.

Reply:
yes, if we cloned the bjro then we `ForgetAbout` & `Dispose` the compressed item (the one we cloned)
see in lines below we use `item.CompressedItem`